src/components/Render.js

```diff
@@ -0,0 +1,11 @@
+import Addbook from './Addbook';
+import Booklist from './Booklist';
+
+const Renderbooks = () => (
+  <>
+    <Booklist />
+    <Addbook />
+  </>
```

Comment:
- [ ] I will implore you to make sure each book card has a progress bar and all other content is present just as it is in the design. I also noticed the ul list-style is present in your design, kindly remove it also.

### Design Template
![image](https://github.com/Roland-Ntwali/Bookstore/assets/86200356/32b59632-17c0-4180-9ff2-3990aaa8c6dc)

### Your Design
![image](https://github.com/Roland-Ntwali/Bookstore/assets/86200356/3092206a-1353-46c6-bf1f-4c9820557e8e)